INSTALL

```diff
@@ -1,5 +1,31 @@
-Installing Iris currently requires you to build it from source. The 
-latest release is available from https://github.com/SciTools/iris.
+You can either install Iris using the conda package manager or from source.
+
+Installing from conda
+---------------------
+
+Iris can be installed using the `conda <http://conda.pydata.org/>`_ package manager.
+
+Iris is available from conda for the following platforms:
+ * Linux 32-bit,
+ * Linux 64-bit,
+ * Mac OSX 64-bit, and
+ * Windows 32-bit and 64-bit.
+
+To install Iris from conda, you must first download and install conda, for example from
+http://conda.pydata.org/miniconda.html.
+
+Once conda is installed, you can install Iris from conda on any platform with the following command::
```

Comment:
There are a number of long (i.e. >79) character lines here which make it hard to read as a text file.